scripts/vnet_route_check.py

```diff
@@ -326,12 +350,17 @@ def main():
     # Don't run VNET routes consistancy logic if there is no VNET configuration
     if not check_vnet_cfg():
         return rc
+    asic_db = swsscommon.DBConnector('ASIC_DB', 0)
+    virtual_router = swsscommon.Table(asic_db, 'ASIC_STATE:SAI_OBJECT_TYPE_VIRTUAL_ROUTER')
+    default_vrf_oid = ""
+    if virtual_router.getKeys() != []:
+        default_vrf_oid = virtual_router.getKeys()[0]
```

Comment:
please make it as a global variable so you don't have to pass it in multiple places and use as "self.xxx"

Reply:
ill do so.

Reply:
done